Packages/Localization/Sources/Localized.swift

```diff
@@ -64,6 +64,18 @@ public enum Localized {
       public static let reserved = Localized.tr("Localizable", "asset.balances.reserved", fallback: "Reserved")
     }
   }
+  public enum AssetTag {
```

Comment:
```suggestion
  public enum AssetTag {
```

Should be Under `Assets.Tag.gainers`